app/components/geoblacklight/index_map_inspect_component.html.erb

```diff
@@ -1,4 +1,10 @@
-<div class='row'>
-  <div class='viewer-information col-sm-12'>
+<div id="<%= Settings.LEAFLET.SIDEBAR_ID || 'table-container' %>" data-sidebar="<%= Settings.LEAFLET.SIDEBAR_ID %>">
```

Comment:
I think I'd rather have the simplicity/readability of a hardcoded/default sidebar ID than whatever this is allowing you to do. Judging from the javascript, if `SIDEBAR_ID` is set, we will find this element, look at its data attribute, and then use that to find another element...which is always itself? I'm not convinced the flexibility is necessary right now.